src/main/java/teammates/ui/controller/FeedbackParticipantFeedbackResponseCommentDeleteAction.java

```diff
@@ -104,4 +101,11 @@ private void appendStatusToAdmin(Long commentId) {
                 + "Deleting feedback response comment: " + commentId + "<br>"
                 + "in course/feedback session: " + courseId + "/" + feedbackSessionName + "<br>";
     }
+
+    private StudentAttributes getStudent() {
+        if (student == null) {
+            student = logic.getStudentForGoogleId(courseId, account.googleId);
```

Comment:
```
    private StudentAttributes getStudent() {
        if (student == null) {
            // branch of student != null is not covered since student is not set elsewhere, but this
            // helps to speed up the process of 'getting' a student so we should leave it here
            student = logic.getStudentForGoogleId(courseId, account.googleId);
        }

        return student;
    }
```
I found this explanation in `FeedbackSubmissionEditPageAction`?

Reply:
Existing code does not mean it is correct. The reason is that the re-assignment breaks the assumption that student is null for logged in user.